packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/dashboard/manage-nsx-edges/dedicatedCloud-datacenter-manage-nsx.routing.js

```diff
@@ -0,0 +1,19 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'app.dedicatedCloud.details.datacenter.details.dashboard.nsx',
+    {
+      resolve: {
+        goBack: /* @ngInject */ (goBackToDashboard) => goBackToDashboard,
```

Comment:
```suggestion
        goBack: /* @ngInject */ (goBackToDashboard) => () => goBackToDashboard,
```

Reply:
@ghyenne , are you sure ? As  `goBackToDashboard` is already a function, the current definition is good for me

Reply:
@darsene No, i'm not sure, but i have the impression that it will try to solve the function directly during the resolve.
Normally this should have been tested, I imagine that the page is displayed when we're not sent directly to the dashboard.

Reply:
Not sure to understand the issue, goBack: /* @ngInject */ (goBackToDashboard) => goBackToDashboard means that when goBack is called, the function goBackToDashboard() is executed.
The second way would make sense if we wanted to modify the behavior of the function or send some parameters, which is not the case